diff --git a/src/tiny-slider.js b/src/tiny-slider.js
index 110b2f4113d77735a727944eec4edd6cf11500bc..b71bd2f39c917a1a9b1245a301b995fc28945fe6 100644
--- a/src/tiny-slider.js
+++ b/src/tiny-slider.js
@@ -1,25 +1,3 @@
-// Object.keys
-if (!Object.keys) {
-  Object.keys = function(object) {
-    var keys = [];
-    for (var name in object) {
-      if (Object.prototype.hasOwnProperty.call(object, name)) {
-        keys.push(name);
-      }
-    }
-    return keys;
-  };
-}
-
-// ChildNode.remove
-if(!("remove" in Element.prototype)){
-  Element.prototype.remove = function(){
-    if(this.parentNode) {
-      this.parentNode.removeChild(this);
-    }
-  };
-}
-
 import { raf } from './helpers/raf.js';
 import { caf } from './helpers/caf.js';
 import { extend } from './helpers/extend.js';
